scripts/docker-entrypoint-charlie.sh

```diff
@@ -12,7 +12,7 @@ echo "Chain Version is ${CHAIN_VERSION}"
   --base-path /tmp/polkadot-chains/charlie \
   --bootnodes /ip4/${ENDPOINT_DEVELOPMENT}/tcp/30333/p2p/${BOOTNODE_NODE_ID_LOCAL} \
```

Comment:
suggest moving the following

scripts/docker-entrypoint-alice.sh
scripts/docker-entrypoint-bob.sh
scripts/docker-entrypoint-charlie.sh
scripts/docker-entrypoint-dave.sh
scripts/docker-entrypoint-eve.sh

into a subfolder called 'local',

scripts/local/docker-entrypoint-alice.sh
scripts/local/docker-entrypoint-bob.sh
scripts/local/docker-entrypoint-charlie.sh
scripts/local/docker-entrypoint-dave.sh
scripts/local/docker-entrypoint-eve.sh

Reply:
ok